pkgs/development/python-modules/ipython/default.nix

```diff
@@ -21,33 +20,22 @@
 , stack-data
 , traitlets
 
-# Test dependencies
+  # Test dependencies
```

Comment:
Please remove the reindentation of the comments.

Reply:
They are `nixpkg-fmt`-ed.